changelogs/fragments/72699-validate-modules-default-for-bools.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- "ansible-test validate-modules - no longer assume that ``default`` for ``type=bool`` options is ``false``, as the default is ``none`` and for some modules, ``none`` and ``false`` mean different things (https://github.com/ansible/ansible/issues/69561)."
```

Comment:
Since the default value for some modules is changing in the argument spec, it should probably be mentioned here so there is documentation for each module change.